### PR TITLE
feature/MTM-39670-Many-Series-within-Measurement-breaks-Measurement-UI

### PR DESCRIPTION
**Pull request by Airdroplet**
* added recommendation for number of series

### CODE REVIEW

File: content/users-guide/device-management-bundle/device-details.md [hunk 1]
```diff
@@ -325,6 +325,8 @@ If a chart contains graphs with different units, one Y-axis is rendered per unit
 
 To see detailed information about the measured values, hover over the chart. A tooltip will be displayed with detailed information on the measurement next to your cursor (the tooltip will "snap" to the closest measurement).
 
+> **Info:** We recommend to not have more than 20 series per measurement. Having too much series for single measurement will greatly decrease performance of single graph and make it harder to review measurements values per series from within device management application. For more detailed data analysis it is better to use data explorer feature of cockpit application.
```
Comment (BeateRixen) [inline content/users-guide/device-management-bundle/device-details.md:328]:
```suggestion
> **Info:** We recommend you to not have more than 20 series per measurement. Having too much series for a single measurement strongly decreases the performance of a single graph and makes it harder to review measurement values per series from within the Device Management application. For a more detailed data analysis, better use the data explorer feature in the Cockpit application.
```

**Review by BeateRixen — APPROVED** [review]

File: content/users-guide/device-management-bundle/device-details.md [hunk 2]
```diff
@@ -325,6 +325,8 @@ If a chart contains graphs with different units, one Y-axis is rendered per unit
 
 To see detailed information about the measured values, hover over the chart. A tooltip will be displayed with detailed information on the measurement next to your cursor (the tooltip will "snap" to the closest measurement).
 
+> **Info:** We recommend you to not have more than 20 series per measurement. Having too much series for a single measurement strongly decreases the performance of a single graph and makes it harder to review measurement values per series from within the Device Management application. For a more detailed data analysis, better use the data explorer feature in the Cockpit application.
```
Comment (kaczynskid-sag) [inline content/users-guide/device-management-bundle/device-details.md:328]:
```suggestion
> **Info:** We recommend you to not have more than 20 series per measurement. Having too many series for a single measurement strongly decreases the performance of a single graph and makes it harder to review measurement values per series from within the Device Management application. For a more detailed data analysis, better use the data explorer feature in the Cockpit application.
```

Comment (BeateRixen) [reply content/users-guide/device-management-bundle/device-details.md:328]: It is "we recommend you to" and not "we recommend to" :-).

Comment (pawel-rynarzewski-sag) [reply content/users-guide/device-management-bundle/device-details.md:328]: What about:
```suggestion
> **Info:** We recommend you to have max. 20 series per measurement for optimal performance and readability of a single graph in Device Management (the graph displays all available series). If you need to display only a few series from the measurement, we advise you to use Data explorer in Cockpit where you can select series to be shown in the graph.
```
?

**Review by pawel-rynarzewski-sag — APPROVED** [review]

**Comment by BeateRixen** [comment]
@Airdroplet Do we still wait for some more approval or can this be merged?

**Comment by Airdroplet** [comment]
@BeateRixen please merge :)

**Comment by BeateRixen** [comment]
> @BeateRixen please merge :)

Should this be grafted to some version?

**Comment by Airdroplet** [comment]
@BeateRixen it can be grafted to all maintenance versions, it is up to you to decide. This documentation describes limitation of feature which being affecting the product from beginning of time.

**Pull request merged**